nilearn/decoding/tests/test_fista.py

```diff
@@ -48,24 +51,37 @@ def test_input_args_and_kwargs():
     y = sig + noise_std * rng.randn(*sig.shape)
     X = np.eye(p)
     mask = np.ones((p,)).astype(bool)
-    alpha = .01
+    alpha = 0.01
     alpha_ = alpha * X.shape[0]
-    l1_ratio = .2
+    l1_ratio = 0.2
     l1_weight = alpha_ * l1_ratio
-    f1 = lambda w: _squared_loss(X, y, w, compute_grad=False)
-    f1_grad = lambda w: _squared_loss(X, y, w, compute_grad=True,
-                                      compute_energy=False)
-    f2_prox = lambda w, l, *args, **kwargs: (_prox_l1(w, l * l1_weight),
-                                             dict(converged=True))
-    total_energy = lambda w: f1(w) + l1_weight * np.sum(np.abs(w))
+
+    def f1(w):
+        return _squared_loss(X, y, w, compute_grad=False)
+
+    def f1_grad(w):
+        return _squared_loss(X, y, w, compute_grad=True, compute_energy=False)
+
+    def f2_prox(w, l, *args, **kwargs):  # noqa: E741
```

Comment:
are `*args` and `**kwargs` really needed here?

Reply:
They might not be used in the function but be needed because some calls to this function specify kwargs which are not w or l, so I'd probably keep *args and **kwargs here.